script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -875,7 +875,7 @@ If nothing else, at least stop when you reach the point where your blood vessels
 [sha:495604819c4d7b1f9f72b1412805a48de18889b1]{
 -- Page 491, Offset 17789.
 -- 「その眼鏡を作ってくれた魔術師には感謝しないとね。
-"You should show some appreciation for the magician that created those glasses for you.
+"You should show some appreciation for the mage that created those glasses for you.
```

Comment:
why not use `magus `instead of `mage `btw?

Reply:
I tend not to use "magus" because a [magus](https://en.wikipedia.org/wiki/Magi) is not the same as a [mage](https://en.wikipedia.org/wiki/Mage) linguistically. I have a feeling this is a translation error stemming from the [CT character](https://en.wikipedia.org/wiki/Characters_of_Chrono_Trigger#Magus) in the first place. It should _probably_ be mage but I have no dog in the TM term correctness race.

According to this tm wiki though, it seems that it should be [magus](https://typemoon.fandom.com/wiki/Magus) in this case.

Reply:
It's just because Magus sounds similar to 魔術師 (Majustushi) and to make it very distinct when 魔法使い (Mahoutsukai / Magican) characters like Zelretch and Aoko, users of Magic, are mentioned to the reader who might assume Mage is short for Magician.

Reply:
![magus](https://user-images.githubusercontent.com/90936434/167528141-1b0f2c47-18c8-4e0c-b740-660d8d30c6b6.PNG)
for reference. then I'll keep it mage since it's been already consistently so.